crates/sui-indexer/README.md

```diff
@@ -0,0 +1,32 @@
+Sui indexer is an off-fullnode service to serve data from Sui protocol, including both data directly generated from chain and derivative data.
+
+## Current architecture Dec 2022 (will change soon)
+![indexer_simple](https://user-images.githubusercontent.com/106119108/209000367-4c7d23d8-fef2-4485-8472-89c31f0e2d62.png)
+
+## Steps to run locally
+### Prerequisites
+- install local [Postgres server](https://www.postgresql.org/download/)
+- install Diesel CLI, you can follow the [Diesel Getting Started guide](https://diesel.rs/guides/getting-started) up to the *Write Rust* section
+- [optional but handy] Postgres client like [Postico](https://eggerapps.at/postico2/), for local check, query execution etc.
+
+### Steps
+1. DB setup
+```sh
+# DB setup, run the following commands from the /sui-indexer folder
+# .env file under /sui-indexer is required for diesel cmds
+# in .env file, DATABASE_URL should point to your local PG server
+# an example is:
+# DATABASE_URL=""postgres://postgres:postgres@localhost/gegao"
+diesel setup
+
+# and then run 
+diesel migration run
+```
+2. cargo run under `/sui-indexer`
+```sh
+cargo run --bin sui-indexer -- --db-url "<LOCAL_DB_URL>" --rpc-client-url "https://fullnode.devnet.sui.io:443"
```

Comment:
Is `LOCAL_DB_URL` same as `postgres://postgres:postgres@localhost/gegao` above?

Reply:
Yes, will change it to DATABASE_URL for consistency then.